toolkits/ob-apim/repository/resources/wso2am-4.2.0-deployment-cds.toml

```diff
@@ -532,6 +532,33 @@ priority = 8
 name = "org.wso2.openbanking.cds.gateway.executors.error.handler.CDSErrorHandler"
 priority = 1000
 
+[[open_banking.gateway.openbanking_gateway_executors.type]]
```

Comment:
Lets remove the unnecessary executors.

Reply:
fixed with c334acc6a04c93c1baf13d40561200c8b105bfbf